mne/decoding/csp.py

```diff
@@ -231,6 +231,10 @@ def transform(self, X):
                 X /= self.std_
         return X
 
+    @copy_doc(TransformerMixin.fit_transform)
+    def fit_transform(self, X, y, **fit_params):
```

Comment:
Oh it's just pydocstyle, not numpydoc -- if you `git grep copy_doc` to see where it's used elsewhere, e.g.:

https://github.com/mne-tools/mne-python/blob/8954c3711b57dc1b90a8079c278384ba684ed17f/mne/io/base.py#L1029-L1035

And cross-ref to the error:
```
        D102: Missing docstring in public method
```

You can see that this in principle should fix it:
```suggestion
    def fit_transform(self, X, y, **fit_params):  # noqa: D102
```